packages/cosmic-swingset/src/import-kernel-db.js

```diff
@@ -145,6 +145,9 @@ export const performStateSyncImport = async (
   // Represent the data in `exportDir` as a SwingSetExporter object.
   /** @type {import('@agoric/swing-store').SwingStoreExporter} */
   const exporter = harden({
+    getHostKV(_key) {
+      return undefined;
```

Comment:
If we wanted to be rounded about this we could expose the blockHeight through this ;)

```suggestion
    getHostKV(key) {
      if (key === 'host.height' && blockHeight) {
        return String(blockHeight);
      } else {
        return undefined;
      }
```

Reply:
per conversation, not worth doing